.babelrc

```diff
@@ -6,4 +6,4 @@
   "plugins": [
       "@babel/plugin-proposal-class-properties"
```

Comment:
you also add @babel/plugin-transform-runtime

Reply:
done